apps/shorturl/kinds/shorturl.cue

```diff
@@ -0,0 +1,16 @@
+package kinds
+
+shorturl: {
+	kind:		"ShortURL"
+	pluralName:	"ShortURLs"
+	group:      "shorturl.grafana.app"
+	current:	"v1alpha1"
```

Comment:
Since versions are no longer contained in kinds, `current` no longer is used, and the analogous feature is `preferredVersion` in the manifest (this defaults to the highest entry in `versions` if not present).